app/helpers/mustache_helper.rb

```diff
@@ -162,6 +162,23 @@ def page_config
     }
   end
 
+  def channel_data
```

Comment:
Assignment Branch Condition size for channel_data is too high. [17.03/15]<br>Method has too many lines. [13/10]